modules/rhel-worker-tag.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+// * machine_management/more-rhel-compute.adoc
+
+
+[id="rhel-worker-tag_{context}"]
+= Tagging a {op-system-base} worker node as owned or shared
+
+A cluster uses the value of the `kubernetes.io/cluster/<clusterid>,Value=(owned|shared)` tag to determine which resources belong to the AWS cluster.
+
+* The `owned` tag value should be added to all AWS components that cannot be shared between two OpenShift clusters.
+* The `shared` tag value should be added to any AWS component of the current OpenShift installation that may be used as part of another OpenShift cluster in the same VPC. For example, subnets in a VPC may use the `kubernetes.io/cluster/<cluster-id>=shared` tag since these are provisioned to install multiple OpenShift clusters in the same VPC.
```

Comment:
I do not think the resources tags and VPCs are relevant, per my understanding, `owned` tags is only for cluster destroy. And for UPI, we created lots of AWS resources for this cluster only, but they do not have `owned` tag.

@staebler  could you please help take a look this part? thanks.

Reply:
The distinction between owned and shared is tied to the lifetime of the resource. A resource that is `owned` should be destroyed as part of destroying the cluster. A resource that is `shared` can continue to exist after the cluster has been destroyed. The `shared` tag does not mean that the resource is used by multiple clusters: It means that the resource is used by the cluster but there is a separate owner of the resource.

Reply:
Excellent! Thanks @yunjiang29 and @staebler for the clarity.

I updated the documentation. You can find an example of it here: https://deploy-preview-31600--osdocs.netlify.app/openshift-enterprise/latest/machine_management/adding-rhel-compute.html#rhel-worker-tag_adding-rhel-compute

Is this clearer/accurate?